BucketInfoModelVersion.md

```diff
@@ -0,0 +1,82 @@
+# BucketInfo Model Version History
```

Comment:
I am a huge fan of sphinx `versionAdded` label and think it would work great here.
Turns out JsDoc has this here https://jsdoc.app/tags-since.html.

```javascript
this._acl = aclInstance;
this._name = name;
this._owner = owner;
this._ownerDisplayName = ownerDisplayName;
this._creationDate = creationDate;
/** JSDOC information here
 * @since v2
 * @type {number}
 */
this._mdBucketModelVersion = mdBucketModelVersion || 0
//etc
```

Could probably remove the markdown entirely and just list it in the javascript too.